docs/src/advanced/generic_types.md

```diff
@@ -108,6 +119,8 @@ struct Foo<T>
 }
 ```
 
+*`where` clauses are still work-in-progress, so some above statements may not be fully implemented*
```

Comment:
Oh I think should actually go at the end of the "Traits Constraints" section instead.